pulumi/volumeReaper/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@shapeshiftoss/volume-reaper",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist node_modules",
+    "build": "tsc",
+    "dev": "ts-node src/index.ts",
+    "format": "prettier -w --config ../../node/.prettierrc --ignore-path ../../node/.prettierignore .",
+    "start": "node dist/index.js"
+  },
+  "dependencies": {
+    "@kubernetes/client-node": "^0.18.1",
+    "commander": "^10.0.0",
+    "figlet": "^1.5.2",
+    "prettier": "^2.8.4",
+    "ts-node": "^10.8.1",
+    "typescript": "^4.7.4"
```

Comment:
these can be moved to `devDependencies` as well. Rule of thumb is all types or any packages only used during local dev (not runtime)